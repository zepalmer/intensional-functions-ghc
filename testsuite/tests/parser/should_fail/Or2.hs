{-# LANGUAGE OrPatterns, PatternSynonyms #-}

module Main where

main = case 3 of
  (one of 4) -> False

g x = case x of
  one of 4, 5 -> False
