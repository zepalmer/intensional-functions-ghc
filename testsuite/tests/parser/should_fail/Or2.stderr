
Or2.hs:6:4: error: [GHC-96152]
    An or-pattern needs at least two alternatives: one of 4

Or2.hs:9:7: error: [GHC-58481] parse error on input ‘of’
